extensions/amp-analytics/0.1/transport.js

```diff
@@ -134,11 +419,11 @@ export function sendRequestUsingIframe(win, request) {
   const iframe = win.document.createElement('iframe');
   setStyle(iframe, 'display', 'none');
   iframe.onload = iframe.onerror = () => {
-    timerFor(win).delay(() => {
+    Transport.timer_.delay(() => {
       removeElement(iframe);
     }, 5000);
   };
-  user().assert(
+  dev().assert(
```

Comment:
Didn't touch the sendRequestUsingIframe() method at all. Note that it is comparing line 141 to line 426, so I suspect that's just the diff tool trying to gain an efficiency when it shouldn't. Want me to change it to dev(), though?